modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -35,32 +35,30 @@ spec:
       excluded:
       - my-namespace
-    podLifetime: 48h                                   <2>
-    thresholdPriorityClassName: my-priority-class-name <3>
-  profiles:                                            <4>
+    podLifetime: 48h                                   <3>
+    thresholdPriorityClassName: my-priority-class-name <4>
+  profiles:                                            <5>
   - AffinityAndTaints
-  - TopologyAndDuplicates                              <5>
+  - TopologyAndDuplicates                              <6>
   - LifecycleAndUtilization
   - EvictPodsWithLocalStorage
   - EvictPodsWithPVC
 ----
---
-<1> Optional: Set a list of user-created namespaces to include or exclude from descheduler operations. Use `excluded` to set a list of namespaces to exclude or use `included` to set a list of namespaces to include. Note that protected namespaces (`openshift-*`, `kube-system`, `hypershift`) are excluded by default.
+<1> Optional: By default, the descheduler does not evict pods. To evict pods, set `mode` to `Automatic`.
+<2> Optional: Set a list of user-created namespaces to include or exclude from descheduler operations. Use `excluded` to set a list of namespaces to exclude or use `included` to set a list of namespaces to include. Note that protected namespaces (`openshift-*`, `kube-system`, `hypershift`) are always excluded.
 +
 [IMPORTANT]
 ====
 The `LowNodeUtilization` strategy does not support namespace exclusion. If the `LifecycleAndUtilization` profile is set, which enables the `LowNodeUtilization` strategy, then no namespaces are excluded, even the protected namespaces. To avoid evictions from the protected namespaces while the `LowNodeUtilization` strategy is enabled, set the priority class name to `system-cluster-critical` or `system-node-critical`.
 ====
-<2> Optional: Enable a custom pod lifetime value for the `LifecycleAndUtilization` profile. Valid units are `s`, `m`, or `h`. The default pod lifetime is 24 hours.
-<3> Optional: Specify a priority threshold to consider pods for eviction only if their priority is lower than the specified level. Use the `thresholdPriority` field to set a numerical priority threshold (for example, `10000`) or use the `thresholdPriorityClassName` field to specify a certain priority class name (for example, `my-priority-class-name`). If you specify a priority class name, it must already exist or the descheduler will throw an error. Do not set both `thresholdPriority` and `thresholdPriorityClassName`.
-<4> Add one or more profiles to enable. Available profiles: `AffinityAndTaints`, `TopologyAndDuplicates`, `LifecycleAndUtilization`, `SoftTopologyAndDuplicates`, `EvictPodsWithLocalStorage`, and `EvictPodsWithPVC`.
-<5> Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates`. Enabling both results in a conflict.
-+
-You can enable multiple profiles; the order that the profiles are specified in is not important.
```

Comment:
This got moved up to be with callout 5. I don't know if that was intentional or not, but it needs to be moved back, because it applies to the whole thing.